zerver/models/streams.py

```diff
@@ -82,23 +81,13 @@ class Stream(models.Model):
     STREAM_POST_POLICY_ADMINS = 2
```

Comment:
These probably need a tombstone comment explaining that they're used for the mappings for legacy API clients, just because otherwise it's a bit weird that they're right here in the middle of the definition of `Realm`, but not used.